eng/common/scripts/job-matrix/Create-PrJobMatrix.ps1

```diff
@@ -144,6 +144,16 @@ function GeneratePRMatrixForBatch {
       }
     }
 
+    if ($matrixConfig.PSObject.Properties['PRBatch']) {
```

Comment:
We should update the matrix and/or unified pr FAQ as well

Reply:
@scbedd is this supposed to be `PRBatch `or `PRBatching`? In our chat it was `PRBatching: false`

Reply:
I started with `PRBatching`, but ben had `PRBatch` and I just accepted the correction without it being said. That being said I prefer `PRBatching` so I'm just going to go with that.